packages/mlmpfr/mlmpfr.4.1.0+bugfix1/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer:   "Laurent Thévenoux <lrnt@thvnx.com>"
+authors:      "Laurent Thévenoux <lrnt@thvnx.com>"
+homepage:     "https://github.com/thvnx/mlmpfr"
+bug-reports:  "https://github.com/thvnx/mlmpfr/issues"
+license:      "LGPL-3.0-only"
+dev-repo:     "git+https://github.com/thvnx/mlmpfr.git"
+build: [
+  ["cc" "mlmpfr_compatibility_test.c" "-lmpfr" "-o" "mlmpfr_compatibility_test"]
+  ["./mlmpfr_compatibility_test"]
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.04"}
+  "dune" {>= "1.11.0"}
+]
+x-ci-accept-failures: ["debian-unstable"]
+depexts: [
```

Comment:
Please remove the depexts and use `conf-mpfr` instead.
(I will do the same for the old packages that need it once this PR is merged)

Reply:
Thanks!